spec.html

```diff
@@ -36049,6 +36069,117 @@ <h1>
             1. Return _index_ + _cp_.[[CodeUnitCount]].
           </emu-alg>
         </emu-clause>
+
+        <emu-clause id="sec-getstringindex" type="abstract operation">
+          <h1>
+            GetStringIndex (
+              _S_: a String,
+              _Input_: a List,
+              _e_: a non-negative integer,
+            )
+          </h1>
+          <dl class="header">
+          </dl>
+          <emu-alg>
+            1. Assert: _Input_ contains the code points of _S_ interpreted as a UTF-16 encoded string.
```

Comment:
```suggestion
            1. Assert: _Input_ contains the code points of StringToCodePoints(_S_).
```

Reply:
I've changed this a bit after I moved the step to derive _Input_ out of CompilePattern and into RegExpBuiltinExec.

Reply:
Are you saying those changes have rendered my suggestion invalid?

Reply:
More that I wasn't certain if you needed to revisit at all, or if I should change the wording in the header:

> `_Input_: a List of characters derived from _S_,`